### PR TITLE
WIP: Cleanup on aisle travis

### PR DESCRIPTION
**Pull request by ericdill**

### CODE REVIEW

**Comment by ericdill** [comment]
Hooray! Looks like datamuxer is python 3.5 compatible too.  This PR should be good to go also.

**Comment by danielballan** [comment]
Much cleaner. Thanks.

File: conda-recipe/meta.yaml [hunk 1]
```diff
@@ -8,26 +8,15 @@ source:
 build:
   number: 0
 
-  # Note that this will override the default build string with the Python
-  #   # and NumPy versions
-  string: {{ environ.get('GIT_BUILD_STR', '') }}_np{{ np }}py{{ py }}
```
Comment (tacaswell) [inline conda-recipe/meta.yaml:13]: Does it still pick up the correct `postN+gHASH` string with out this?

Comment (ericdill) [inline conda-recipe/meta.yaml:13]: I am not sure

**Pull request merged**